### DISCUSSION TITLE
Ubuntu安装测试时报Schema error

### DISCUSSION THREAD

**Discussion by Yuxi-Duan-PKU** [MaaAssistantArknights/MaaAssistantArknights#9662] [Q&A]
我尝试在Ubuntu上安装MAA（MAA-v5.4.0-linux-x86_64），但在测试 'sample.py’ 时，显示“连接成功”后会报一连串的Schema error，看起来似乎是在schema中register时发生了错误。我检查了报错中显示的文件夹/路径，但它们并不存在。（我使用的是Waydroid模拟器）
请问是否有人能够帮助我解决这个问题？我将不胜感激！
![Screenshot from 2024-07-12 18-37-47](https://github.com/user-attachments/assets/d2f37470-581c-4596-82af-600d3b14f772)

**Comment by wangl-cc** [comment]
Duplicate #9618
